docs/src/main/sphinx/installation/cli.rst

```diff
@@ -391,14 +394,46 @@ press :kbd:`Enter`.
 By default, you can locate the Trino history file in ``~/.trino_history``.
 Use the ``TRINO_HISTORY_FILE`` environment variable to change the default.
 
+Batch mode
+----------
+
+Running the Trino CLI with the ``--execute``, ``--file``, or passing queries to
+the standard input uses the batch (non-interactive) mode. In this mode
+the CLI does not report progress, and exits after processing the supplied
+queries. Results are printed in ``CSV`` format by default. You can configure
+other formats and redirect the output to a file.
+
+The following options are available to further configure the CLI in batch
+mode:
+
+.. list-table::
+  :widths: 40, 60
+  :header-rows: 1
+
+  * - Option
+    - Description
+  * - ``--execute=<execute>``
+    - Execute specified statements and exit.
+  * - ``-f``, ``--file=<file>``
+    - Execute statements from file and exit.
+  * - ``--ignore-errors``
+    - Continue processing in batch mode when an error occurs. Default is to
+      exit immediately.
+  * - ``--output-format=<format>``
+    - Specify the format to use for printing query results.
```

Comment:
Link to `Output formats` section?

Reply:
I had a link here but @mosabua suggested a change without it. Should I add it back?

Reply:
IMO it helps future proof since sections may get added or reordered over time and we won't notice then. The link doesn't hurt I think.

Reply:
Done! I'm posting a screenshot to make sure I actually tested the changes :D
![image](https://user-images.githubusercontent.com/795177/165246829-eff5d140-6ea0-448f-89e8-db01c6c6b08d.png)